web/modules/custom/demo_content/demo_content.install

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/**
+ * @file
+ * Includes demo content for the joinup project.
+ */
+
+use Drupal\file\Entity\File;
+use Drupal\rdf_entity\Entity\Rdf;
+
+/**
+ * Implements hook_install().
+ */
+function demo_content_install() {
+  demo_content_create_demo_content();
+}
+
+/**
+ * Creates demo content for the joinup project.
+ */
+function demo_content_create_demo_content() {
+  $asset_path = \Drupal::moduleHandler()->getModule('demo_content')->getPath() . '/includes/images';
+  foreach (demo_content_get_collection_data() as $values) {
+    if (!empty($values['field_ar_logo'])) {
+      $file_path = rtrim(realpath($asset_path), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . $values['field_ar_logo'];
+      if (!is_file($file_path)) {
+        throw new \Exception("File '{$values['field_ar_logo']}' was not found in file path '$asset_path'.");
+      }
+
+      $uri = file_unmanaged_copy($file_path, 'public://' . $values['field_ar_logo']);
+      $file = File::create(['uri' => $uri]);
+      $values['field_ar_logo'] = [$file];
+    }
+
+    $values['rid'] = 'collection';
```

Comment:
Should we move this rid value to the collection data, like it is done for solution?